recipes/y-py/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "y-py" %}
+{% set version = "0.1.3" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/yjs/y-crdt/archive/v{{ version }}.tar.gz
+  sha256: 74c2ad813befa7e7a08658cfaa5d6ff377343b76f3014aa91cf05bce7bfa7c3a
+
+build:
+  number: 0
+  skip: true  # [not linux]
+
+requirements:
+  build:
+    - gcc
```

Comment:
this should probably be `- compiler("c")`

Reply:
Also there is `- {{ compiler('rust') }}` that would be much preferred

Reply:
This was needed for the linker.